RecoMET/METProducers/src/HcalNoiseInfoProducer.cc

```diff
@@ -428,39 +623,61 @@ HcalNoiseInfoProducer::filldigis(edm::Event& iEvent, const edm::EventSetup& iSet
   // get total charge in calibration channels
   if(hCalib.isValid() == true)
   {
+
+     // store the data from the lasermon fibers
+     std::map<int, std::vector<int> > lasmon_adcs;
```

Comment:
it looks like these can be vectors instead of maps
std::vector<std::vector<int> > lasmon_capids(laserMonDetTypeList_.size(), std::vector<int>());

Reply:
Yes, I agree.  The map was intended to be more flexible, but it ends up that we just loop in order